EIPS/eip-1845.md

```diff
@@ -0,0 +1,64 @@
+---
+eip: <to be assigned>
+title: Fork Name Standards
+author: (winsvega) <dimitry@ethereum.org>
```

Comment:
The name of the author does not need to be between parenthesis ( )

So it would either be:

`DimitryKh <dimitry@ethereum.org>`

or 

`DimitryKh (@winsvega)`